packages/site/cypress.config.ts

```diff
@@ -1,5 +1,5 @@
 import { defineConfig } from "cypress";
-import replay from "@replayio/cypress";
+const { plugin: replayPlugin } = require("@replayio/cypress");
```

Comment:
This one should work too. I just prefer to avoid mixing CJS/ESM in a single file ;p
```suggestion
import { plugin as replayPlugin } from "@replayio/cypress";
```

If the previous has worked (a default import) then it might be a fallout from my recent changes in the CLI. I'll try to recheck if I changed the behavior around this cause if I did it wasn't intentional